Maquetado/scss/styles.scss

```diff
@@ -1,3 +1,3 @@
 .title {
-    color: red;
-  }
\ No newline at end of file
+  color: red;
+}
```

Comment:
If you want, configure the vscode for auto-apply EOF.

Reply:
Done, thank you!